free-courses-hi.md

```diff
@@ -46,3 +48,10 @@
 ### Python
 
 * [Python Tutorial For Beginners | Hindi (With Notes)](https://www.youtube.com/playlist?list=PLu0W_9lII9agICnT8t4iYVSZ3eykIAOME) - Harry
+
+
+### Ruby
+
+#### Ruby on Rails
+
+* [Ruby on Rails 5 Tutorial(Hindi)](https://www.youtube.com/watch?v=Wfd4ZIBMg7w&list=PLgPJX9sVy92yV7Qt6_8ElC9paGWdtdIbb) - CS Geeks
```

Comment:
https://www.youtube.com/playlist?list=PLgPJX9sVy92yV7Qt6_8ElC9paGWdtdIbb

Reply:
Sorry about that. ^-^'